Shopist/ShopistUITests/ShopistUITests.swift

```diff
@@ -6,15 +6,20 @@
 
 import XCTest
 
-let targetSessionCount = 10
+let targetSessionDuration: TimeInterval = 27.5 * 60.0 // 25mins
```

Comment:
```suggestion
let targetSessionDuration: TimeInterval = 27.5 * 60.0 // 27.5mins
```

Reply:
👍 also i just noticed `targetSessionDuration` is misleading, it should be `targetRunDuration`